site/list

```diff
@@ -0,0 +1,14 @@
+images from david
+- How to migrate from Amazon Cognito
+- We'll see you at Codemash
+- Extending FusionAuth roles with Cerbos
+
+do content calendar
+
+interesting conferences
+
+reach out to podcasts
+
+plan for 'how to make things easier'
+
```

Comment:
I don't think you want to check this in?

Reply:
Oops, not sure how that got in there. I'll remove!